ldc-config/ldc2.conf

```diff
@@ -10,6 +10,9 @@ default:
         "-I%%ldcbinarypath%%/../include/d/ldc",
         "-I%%ldcbinarypath%%/../include/d",
         "-L-L%%ldcbinarypath%%/../lib",
+        "-L-L%%ldcbinarypath%%/../lib32",
+        "-L-L%%ldcbinarypath%%/../lib64",
+        "-L--no-warn-search-mismatch",
```

Comment:
There have been some changes to the config file (separation into `switches` and `post-switches` as well as new `rpath`).

Reply:
Is that for the compiler in general, or just multilib builds?  I'll check out what is in the current packages.